### ISSUE TITLE
Update README badges, setup.cfg to state Python 3.12 is supported

### ISSUE THREAD

**Issue by eduardo-rodrigues** [scikit-hep/hepstats#161]
Indeed I see that at least these 2 files only mention up to 3.11. This seems to have slipped from PR https://github.com/scikit-hep/hepstats/pull/156.

**Comment by jonas-eschle** [comment]
yes, indeed, thanks, I've updated it. The badges are weird though, the conda version is clearly newer, also when clicking on it

**Comment by eduardo-rodrigues** [comment]
This has been sorted a while back, in fact.
